ptp/protocol/ptp4l.go

```diff
@@ -65,6 +82,43 @@ type TimeStatusNPTLV struct {
 	GMIdentity                 ClockIdentity
 }
 
+// PortServiceStats is a ptp4l struct containing counters for different port events, which we added in linuxptp cfbb8bdb50f5a38687fcddccbe6a264c6a078bbd
+type PortServiceStats struct {
+	AnnounceTimeout       uint64 `json:"ptp.servicestats.announce_timeout"`
+	SyncTimeout           uint64 `json:"ptp.servicestats.sync_timeout"`
+	DelayTimeout          uint64 `json:"ptp.servicestats.delay_timeout"`
+	UnicastServiceTimeout uint64 `json:"ptp.servicestats.unicast_service_timeout"`
+	UnicastRequestTimeout uint64 `json:"ptp.servicestats.unicast_request_timeout"`
+	MasterAnnounceTimeout uint64 `json:"ptp.servicestats.master_announce_timeout"`
+	MasterSyncTimeout     uint64 `json:"ptp.servicestats.master_sync_timeout"`
+	QualificationTimeout  uint64 `json:"ptp.servicestats.qualification_timeout"`
+	SyncMismatch          uint64 `json:"ptp.servicestats.sync_mismatch"`
+	FollowupMismatch      uint64 `json:"ptp.servicestats.followup_mismatch"`
+}
+
+// PortServiceStatsNPTLV is a management TLV added in linuxptp cfbb8bdb50f5a38687fcddccbe6a264c6a078bbd
+type PortServiceStatsNPTLV struct {
+	ManagementTLVHead
+
+	PortIdentity     PortIdentity
+	PortServiceStats PortServiceStats
+}
+
+// MarshalBinary converts packet to []bytes
+func (p *PortServiceStatsNPTLV) MarshalBinary() ([]byte, error) {
```

Comment:
Please unittest this otherwise it will damage the coverage.

Reply:
It is already covered by newly added tests